Essentials/src/main/resources/config.yml

```diff
@@ -666,6 +666,11 @@ jail-online-time: false
 # Set to 0 for no timeout.
 tpa-accept-cancellation: 120
 
+# The maximum number of simultaneous tpa requests that can be pending for any player.
+# Once past this threshold, old requests will instantly time out.
+# Defaults to 15.
+tpa-max-requests: 15
```

Comment:
```suggestion
# The maximum number of simultaneous TPA requests that can be pending for any player.
# Once past this threshold, old requests will instantly time out.
# Defaults to 15.
tpa-max-requests: 15
```

This is quite a high default and I'm pretty sure vanilla clients won't show this many requests in command suggestions. Somewhere between 5-10 might be a better default?

Reply:
5